src/components/new-request/search.vue

```diff
@@ -571,10 +571,11 @@ export default class Search extends Mixins(CommonMixin) {
   }
 
   get showCompanyRadioBtn (): boolean {
-    if (!this.isBcCcCrUl) {
+    const showButton = this.isBcCcCrUl && !(this.isSupportSociety() && this.getEntityTypeCd === EntityType.SO)
```

Comment:
Please add a short comment to explain the business logic here.

Alternately, you could break up the logic into several, well-named variables.